src/App/Infrastructure/Ui/Assets/scss/app/_variables/_z-index.scss

```diff
@@ -0,0 +1,15 @@
+// This file is part of the Php DDD Standard project.
+//
+// Copyright (c) 2017 LIN3S <info@lin3s.com>
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+//
+// @author Beñat Espiña <benatespina@gmail.com>
+
+$z-index-1: 1;
+$z-index-5: 5;
+$z-index-50: 50;
+$z-index-100: 100;
+$z-index-200: 200;
+$z-index-900: 900;
```

Comment:
This is not very useful, `$z-index-header` its okey to place layers above components but having z-index-<number> does not give any semantic value